docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,294 @@
+---
+sidebar_label: 'Manage Expectation Suites'
+title: 'Manage Expectation Suites'
+description: Create and manage sets of Expectations in Python with GX Core.
+---
+
+An Expectation Suite contains a group of Expectations that describe the same set of data.  All of the Expectations that you apply to your data will grouped into Expectation Suites.
+
+## Prerequisites
+
+This guide assumes you have:
+
+- Installed Python.
+- Installed the GX Core library.
```

Comment:
```suggestion
- The GX Core library.
```
Link?

Reply:
It needs to be installed before it can be imported.  The installation is the `pip install great_expectations` step, not the `import great_expectations as gx` step.  I'll be replacing this line with a link to the appropriate guide once I've revised the installation and setup guides from GX OSS for GX Core.